test/support/embedded_model.ex

```diff
@@ -0,0 +1,21 @@
+defmodule ValidField.Support.EmbeddedModel do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  alias ValidField.Support.EmbeddedModel
+
+  embedded_schema do
+    field(:street, :string)
+    field(:city, :string)
+    field(:postcode, :string)
+
+    timestamps()
+  end
+
+  def changeset(%EmbeddedModel{} = embedded_model, params \\ :empty) do
```

Comment:
I think the preference here would be to use `__MODULE__` rather than the alias

Reply:
I preference an explicit name, as you immediately see, what it's about.  Here's the `__MODULE__` version:

```suggestion
  def changeset(%__MODULE__{} = embedded_model, params \\ :empty) do
```